docs/DeveloperGuide.md

```diff
@@ -954,27 +958,27 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 <br>
 
-**Use case 16: Launch policies window**
+**Use case 16: View insurance policies of selected client**
 
 **MSS**
 
 1.  User requests to display policies associated with a selected client.
 
-2.  Window showing all policies associated with this client is launched.
+2.  ClientBook shows all policies associated with this client.
 
     Use case ends.
 
 **Extensions**
 
 * 1a. The selected client has no policies.
 
-    * 1a1. Window launched shows that selected client has no associated policies.
+    * 1a1. ClientBook shows message indicating that no policies are associated with the selected client.
 
       Use case ends.
 
-* 1a. The given index is invalid.
+* 1b. The given arguments are invalid.
```

Comment:
It could be one or more of the given arguments are invalid.

Reply:
What do you mean by this?

Reply:
The current sentence could mean that only when all arguments are invalid then it give error.